packages/components/src/popover/index.js

```diff
@@ -273,14 +274,27 @@ const Popover = ( {
 				contentRect.current = contentRef.current.getBoundingClientRect();
 			}
 
+			let boundaryElement;
+
+			if ( __unstableBoundaryParent ) {
+				boundaryElement = containerRef.current.closest( '.popover-slot' ).parentNode;
```

Comment:
Is this a parent element? I mean can we use React context to leverage this instead?

Reply:
Store the ref in context?

Reply:
yes, that's the idea.

Reply:
For some reason, I cannot get it to work. The context I set around the slot is not available in the popover.

Reply:
I think it's not a parent. Let's keep it that way then.